Cython/Debugger/libcython.py

```diff
@@ -776,8 +776,8 @@ def _break_pyx(self, name):
 
         if (cython_module.filename, lineno) in cython_module.lineno_cy2c:
             c_lineno = cython_module.lineno_cy2c[cython_module.filename, lineno]
-            breakpoint = '%s:%s' % (cython_module.c_filename, c_lineno)
-            gdb.execute('break ' + breakpoint)
+            breakpnt = '%s:%s' % (cython_module.c_filename, c_lineno)
```

Comment:
Wanted to avoid shadowing the breakpoint keyword, which didn't exist when this code was first written